code/game/objects/items/weapons/RCD.dm

```diff
@@ -131,6 +131,61 @@
 	GLOB.rcd_list -= src
 	return ..()
 
+/obj/item/rcd/suicide_act(mob/living/user)
+	user.Immobilize(10 SECONDS) // You cannot move.
+	flags |= NODROP				// You cannot drop. You commit to die.
+	var/turf/suicide_tile = get_turf(src)
+	if(mode == MODE_DECON && checkResource(5, user))	// Same cost as deconstructing a wall.
+		user.visible_message("<span class='suicide'>[user] points [src] at [user.p_their()] chest and pulls the trigger.  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+		playsound(loc, 'sound/machines/click.ogg', 50, TRUE)
+		to_chat(user, "Deconstructing User...")
```

Comment:
No span class?

Reply:
All of the other construction/deconstruction dialog lacks span classes.

Should I give all of them spans or..?

Reply:
Only here is good.